.solcover.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
     norpc: true,
-    testCommand: 'node --max-old-space-size=4096 ../node_modules/.bin/truffle test --network coverage',
+    copyPackages: ['@aragon/os'],
```

Comment:
As far as I can tell, the only reason we're importing aragonOS in this package is for `AbstractENS`.

Maybe we can just include that interface directly, rather than having aragonOS as a dependency?